nuxeo-platform-video-core/src/main/java/org/nuxeo/ecm/platform/video/service/VideoServiceImpl.java

```diff
@@ -191,7 +191,9 @@ public VideoConversionStatus getProgressStatus(String repositoryName, String doc
         WorkManager workManager = Framework.getLocalService(WorkManager.class);
         Work work = new VideoConversionWork(repositoryName, docId, conversionName);
         State state = workManager.getWorkState(work.getId());
-        if (state == State.SCHEDULED) {
+        if (state == null) { // DONE
+            return null;
```

Comment:
Why do you return `null` and not a `VideoConversionStatus` like for other code paths? Is the caller prepared to get a `null`? The Javadoc should explain what it means in that case.